poms/bom/pom.xml

```diff
@@ -5998,29 +5998,29 @@
                 <artifactId>jsoup</artifactId>
                 <version>${jsoup.version}</version>
             </dependency>
-            <dependency>
-                <groupId>org.kie.soup</groupId>
-                <artifactId>kie-soup-commons</artifactId>
-                <version>${optaplanner.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.kie.soup</groupId>
-                <artifactId>kie-soup-project-datamodel-commons</artifactId>
-                <version>${optaplanner.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>org.mvel</groupId>
-                <artifactId>mvel2</artifactId>
-                <version>${mvel2.version}</version>
-            </dependency>
+             <dependency>
```

Comment:
Minor nit - few dependencies here got some additional whitespace added to them, so they are not aligned nicely anymore.

Reply:
yes!! thanks